.github/workflows/ci-build.yml

```diff
@@ -28,3 +28,10 @@ jobs:
         run: yarn install --frozen-lockfile
       - name: Build app
         run: yarn build
+      - name: Upload build artifacts
+        # Upload build artifacts on push event.
+        if: github.event_name == 'push'
+        uses: actions/upload-artifact@v3
+        with:
+          name: explorer-build
```

Comment:
I would perhaps name this: `oasis-explorer-<short-git-commit-hash>` so the artifact users download will be:
```
oasis-explorer-<short-git-commit-hash>.zip
```